mmdet/core/bbox/coder/delta_xywh_bbox_coder.py

```diff
@@ -199,9 +199,25 @@ def delta2bbox(rois,
     x2 = gx + gw * 0.5
     y2 = gy + gh * 0.5
     if clip_border and max_shape is not None:
-        x1 = x1.clamp(min=0, max=max_shape[1])
-        y1 = y1.clamp(min=0, max=max_shape[0])
-        x2 = x2.clamp(min=0, max=max_shape[1])
-        y2 = y2.clamp(min=0, max=max_shape[0])
+        if torch.onnx.is_in_onnx_export():
+            min = torch.tensor(0, dtype=torch.float32)
```

Comment:
Try use meaningful variable name for `min`, `max1`, `max2`. For example: `zero`, `width`, 'height'.

Reply:
sure,i'll update it